### PR TITLE
Set sorts on Splits when adding them to list.

### LINKED ISSUE #376: Clicking on split in not fully ready visualization causes an app to crash
**Reproduce**

- open turnilo
- choose line chart component - you get a message to choose continuous split
- add NON-continuous split first
- click on split to see its toolbar
Effect: the app crashes ;)

**What is happening?**

Every visualization in so called `manifest` has logic that sets different properties on splits (depending on specifics of given visualization).

But this happens only after all splits required by visualization have been added - before that splits wait in "limbo" state which results in buggy behavior.

This problem will become more apparent in #372 which introduces first visualization to require 2 splits. After first one is added, if user clicks on this split, an app will crash.

### PR DESCRIPTION
**Pull request by adrianmroz**
Closes #376

### CODE REVIEW

File: src/common/manifests/table/table.ts [hunk 1]
```diff
@@ -67,7 +45,7 @@ const rulesEvaluator = visualizationDependentEvaluatorBuilder
       autoChanged = true;
     }
 
-    return autoChanged ? Resolve.automatic(6, { splits: newSplits }) : Resolve.ready(isSelectedVisualization ? 10 : 8);
+    return autoChanged ? Resolve.automatic(6, { splits: newSplits }) : Resolve.ready(isSelectedVisualization ? 10 : 6);
```
Comment (adrianmroz) [inline src/common/manifests/table/table.ts]: Yup, it's change from one magic number to different. 🌈

File: src/common/manifests/bar-chart/bar-chart.ts [hunk 2]
```diff
@@ -39,37 +38,7 @@ const rulesEvaluator = visualizationDependentEvaluatorBuilder
 
     const newSplits = splits.update("splits", splits => splits.map((split: Split) => {
       const splitDimension = dataCube.getDimension(split.reference);
-      const sortStrategy = splitDimension.sortStrategy;
```
Comment (mpodlasin) [inline src/common/manifests/bar-chart/bar-chart.ts]: God, this is beautiful... 😮

File: src/common/manifests/table/table.ts (hunk 1 above)
Comment (mpodlasin) [reply src/common/manifests/table/table.ts]: Makes sense...

File: src/common/models/essence/essence.ts [hunk 3]
```diff
@@ -487,23 +487,46 @@ export class Essence extends ImmutableRecord<EssenceValue>(defaultEssence) {
...
+    const { dataCube, series } = this;
+    const dimension = dataCube.getDimension(split.reference);
+    const { sortStrategy, name, kind } = dimension;
+    if (sortStrategy === "self" || sortStrategy === name) {
+      return new DimensionSort({ reference: name, direction: SortDirection.ascending });
+    }
+    if (sortStrategy && series.hasMeasureSeries(sortStrategy)) {
+      return new SeriesSort({ reference: sortStrategy, direction: SortDirection.descending });
+    }
+    if (kind === "time") {
```
Comment (adrianmroz) [inline src/common/models/essence/essence.ts]: Just "time" or continuous? I vote for time. Numeric dimensions with limits are tricky.

Comment (piter75) [reply src/common/models/essence/essence.ts]: IMHO only time is so specific to set default sort ascending by dimension.

**Pull request merged**